recipes/cloud-data-connector/meta.yaml

```diff
@@ -22,9 +22,9 @@ requirements:
     - pip
   run:
     - python >=3.9,<3.11
-    # - azureml >=0.2.7 # not available in conda
+    - azureml >=0.2.7
     - azure-ai-ml >=1.4.0 
-    # - azureml-core >=1.49.0 # not available in conda
+    # - azureml-core >=1.49.0 # There is no sdist package on pypi
```

Comment:
Is it a mandatory dependency?

Reply:
It kinds of defeats the purpose of creating a conda-package. Do they publish a tagged source on GitHub?

Reply:
1. pip sends to a microsoft page: https://learn.microsoft.com/en-us/python/api/overview/azure/ml/?view=azure-ml-py
2. that microsoft page sends to a github page through `release notes`: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/CHANGELOG.md
that last page refers to `azure-ai-ml` which is a dependecy that exists in pip: https://pypi.org/project/azure-ai-ml/
`azure-ai-ml` is already a `cloud-data-connector` dependency. Let me do some tests to see if I can remove `azureml-core`

Reply:
It seems that we really need `azureml-core`. 
I understand that is not the ideal scenario, but since I didn't find any source from where to build the package, it seems that for now, we will need to install it from pypi. 
I already contacted microsoft team to push into adding source distribution to PyPI.

Reply:
@ocefpaf `azureml-core` is only needed for certain non-core functionalities of `data-connector`. If we build it without it, the package will still work. A solution proposal is to comment in the wiki to add that package only if that certain functionality is needed. Do you agree with this?

Reply:
I'm not a user of these packages to answer this but I trust your judgement. If users complain we can pull the package.

Please open an issue in the feedstock once it is created b/c those are easier to find than wikis.

Reply:
That race condition is "fine" and all you need to do is to restart the CIs. The issue was about the lack of `azureml-core`, no? I confess I lost track at this point.

Reply:
Yes, you are correct. Let me write a quick summary:
1. `azureml` is already in conda-forge
2. `cloud-data-connector` needs to restart CIs. I opened an issue to restart rendering.
3. `azureml-core` is not available and will limit some of `cloud-data-connector` functionalities.
It's just that I wasn't sure how to approach the issue about `azureml-core`. Should I just describe the issue and leave it open?

Reply:
Yep. Open an issue for 3 and let's hope upstream will make the sdist available one day.